docs/applications/cloud-storage/install-nextcloud-13.md

```diff
@@ -0,0 +1,122 @@
+---
+author:
+  name: Angel
+  email: docs@linode.com
+description: "Nextcloud is an open source solution to hosting your own content online. In addition to the total control users gain over their own files, Nextcloud offers customizable security features that allow the user to take control of sharing and access privileges."
+og_description: "Nextcloud 13 Brings Improved UI, video and text chat, and end-to-end encryption wrapped into a cloud storage platform. This guide will walk you through installing Nextcloud 13 using Docker."
+keywords: ["nextcloud", "cloud", "open source hosting", "video chat"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2017-03-14
+modified: 2018-03-07
+modified_by:
+  name: Angel
+title: 'Install NextCloud 13'
```

Comment:
Install and Nextcloud Talk

Also add styling guideline for `Nextcloud` - not `NextCloud`